### DISCUSSION TITLE
How to localize service email.

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#11552] [General] [answered]
Hi guys!
Sorry if I asking again :D
But this time I don't know how to send a service mail (for example reset password) in the correct locale.

If I am in the Italian website I would like to receive the correct email. Same if I'm in the uk version.

But I think `{{#if (compare user.language_code "==" "it")}}` doesn't work and in the mutation params I didn't see anything concerning locale.

Thanks in advance and kind regards,

Davide

**Comment by korycins** [comment, marked as answer]
Hey @DidoMarchet ,

Just to confirm. Are you sure that you set up the languageCode correctly?
For account register, you can pass `it` as an enum value for `languageCode` field of `AccountRegister`.

Just checked inside some custom email templates and we are using there `compare` without a problems. The syntax as below (I've changed the paramaters to match your case).
```
{{#if (compare user.language_code "==" "it")}}
  <<somtehing>>
{{/if}}
```

If you still have an issue, I would recommend to print (for debug purposes) the `user.language_code` inside the email.  Let us know about the result.

Also just letting you know, that you can use `NOTIFY` webhook to build your own Saleor Email App, or create one that will work as a proxy between Saleor and your email provider.
